unified-codes/tools/scripts/dgraph/data/v2/schema.gql

```diff
@@ -0,0 +1,83 @@
+type Property {
+  type
+  value
+}
+
+type Category {
+  code
+  name
+  children
+  properties
+}
+
+type Product {
+  code
+  name
+  combines
+  children
+  properties
+}
+
+type Route {
+  code
+  name
+  children
+  properties
+}
+
+type DoseForm {
+  code
+  name
+  children
+  properties
+}
+
+type DoseQualification {
+  code
+  name
+  children
+  properties
+}
+
+type DoseStrength {
+  code
+  name
+  children
+  properties
+}
+
+type DoseUnit {
+  code
+  name
+  children
+  properties
+}
+
+type InnerPack {
```

Comment:
```
export interface IPackImmediateEntity extends IEntity {
  code: string;
  name: string;
  type: EEntityType.PackImmediate;
  children?: IPackImmediateEntityChild[];
  properties?: IPackImmediateEntityProperty[];
}
```

Hmm, type is currently `PackImmediateEntity`, do we want to go with `InnerPack` or `PackImmediate` (or some other variation of)?

Reply:
I think in this instance `PackImmediate` is probably better than `InnerPack`.
It's called `Immediate Packaging` in the spreadsheet but that's a bit long...

Reply:
Yeah I say go with `PackImmediate`!

Reply:
2a792f9cf